schema/manifest_schema.json

```diff
@@ -15,7 +15,8 @@
         "seq_attrib" : { "$ref" : "#/definitions/file_info" },
         "genome" : { "$ref" : "#/definitions/file_info" },
         "functional_annotation" : { "$ref" : "#/definitions/file_info" },
-        "agp" : { "$ref" : "#/definitions/agp_info" }
+        "agp" : { "$ref" : "#/definitions/agp_info" },
+        "events" : { "$ref" : "#/definitions/file_info" }
```

Comment:
This is the most annoying thing for me from JSON: not being able to finish the last element with `,` so the new additions only cover new lines.

Reply:
Good idea to add a json checker to our CI/CD then?

Reply:
Yes, this is already planned once I finally sit down and push on the CI/CD documentation.